app/page.tsx

```diff
@@ -4,27 +4,29 @@ import AnimatedDivider from '../components/AnimatedDivider'
 import AnimatedHero from '../components/AnimatedHero'
 import AnimatedFireworks from '../components/AnimatedFireworks'
 import Image from 'next/image'
+import React from 'react'
+import { Title } from '@/components/title/Title'
 
 export default function Home() {
   return (
-    <div className={'columns'}>
+    <div className="flex m-[0.75rem]">
       <div className={styles.wendy}>
         <Image src={'/svg/JZ2023-Character-Wendy-cropped.svg'} width={300} height={300} alt={'f'} />
       </div>
 
-      <section className={`has-text-centered`}>
+      <section className="text-center">
         <div className="mb-4">
           <AnimatedHero />
         </div>
 
-        <div>
+        <div className="flex justify-center">
           <AnimatedDivider />
           <AnimatedFireworks />
         </div>
 
-        <h1 className={`title ${styles.fontAndColor} mt-4`}>
+        <Title className={`mt-4 font-['Noto Serif', 'Open Sans', 'serif'] `}>
```

Comment:
Jeg tenker at det er lurt om vi kan definere fontene ett sted i koden, slik at det er lettere å følge og mindre rom for feil

Reply:
ref. kommentaren min på "subtitle"